manifests/pe_metric.pp

```diff
@@ -55,6 +55,7 @@
     ensure  => $metric_ensure,
     user    => 'root',
     hour    => '2',
+    minute  => '0',
```

Comment:
I think I've made this mistake with pretty much every cron job I've ever made with Puppet.